doc/cpp/morphology.rst

```diff
@@ -224,14 +224,14 @@ From morphologies to points and segments
 
 The :cpp:type:`morphology` class has the ``branch_segments`` method for
 returning a vector of :cpp:type:`msegment` objects that describe the geometry
-of that branch. However, determining the position in space of an
+of that branch. However, determining the position in the space of an
```

Comment:
The 'the' here feels wrong, to me. We are saying 
X determines the position and orientation of Y in space. 
'Space' as in the abstract notion of 3D Eulerian space.